src/pages/workspace/WorkspaceMoreFeaturesPage.tsx

```diff
@@ -0,0 +1,206 @@
+            titleTranslationKey: 'workspace.moreFeatures.workflows.title',
+            subtitleTranslationKey: 'workspace.moreFeatures.workflows.subtitle',
+            isActive: policy?.areWorkflowsEnabled ?? false,
+            pendingAction: policy?.pendingFields?.areWorkflowsEnabled,
+            action: (isEnabled: boolean) => {
+                Policy.enablePolicyWorkflows(policy?.id ?? '', isEnabled);
+            },
+        },
+    ];
+
+    const organizeItems: Item[] = [
+        {
+            icon: Illustrations.FolderOpen,
+            titleTranslationKey: 'workspace.moreFeatures.categories.title',
+            subtitleTranslationKey: 'workspace.moreFeatures.categories.subtitle',
+            isActive: policy?.areCategoriesEnabled ?? false,
+            pendingAction: policy?.pendingFields?.areCategoriesEnabled,
+            action: (isEnabled: boolean) => {
+                Policy.enablePolicyCategories(policy?.id ?? '', isEnabled);
+            },
+        },
+        {
+            icon: Illustrations.Tag,
+            titleTranslationKey: 'workspace.moreFeatures.tags.title',
+            subtitleTranslationKey: 'workspace.moreFeatures.tags.subtitle',
+            isActive: policy?.areTagsEnabled ?? false,
+            pendingAction: policy?.pendingFields?.areTagsEnabled,
+            action: (isEnabled: boolean) => {
+                Policy.enablePolicyTags(policy?.id ?? '', isEnabled);
+            },
+        },
+        {
+            icon: Illustrations.Coins,
+            titleTranslationKey: 'workspace.moreFeatures.taxes.title',
+            subtitleTranslationKey: 'workspace.moreFeatures.taxes.subtitle',
+            isActive: policy?.tax?.trackingEnabled ?? false,
+            pendingAction: policy?.pendingFields?.tax,
+            action: (isEnabled: boolean) => {
+                Policy.enablePolicyTaxes(policy?.id ?? '', isEnabled);
+            },
+        },
+        {
+            icon: Illustrations.Pencil,
+            titleTranslationKey: 'workspace.moreFeatures.reportFields.title',
+            subtitleTranslationKey: 'workspace.moreFeatures.reportFields.subtitle',
+            isActive: policy?.areReportFieldsEnabled ?? false,
+            pendingAction: policy?.pendingFields?.areReportFieldsEnabled,
+            action: (isEnabled: boolean) => {
+                Policy.enablePolicyReportFields(policy?.id ?? '', isEnabled);
+            },
+        },
+    ];
+
+    const integrateItems: Item[] = [
+        {
+            icon: Illustrations.Accounting,
+            titleTranslationKey: 'workspace.moreFeatures.connections.title',
+            subtitleTranslationKey: 'workspace.moreFeatures.connections.subtitle',
+            isActive: policy?.areConnectionsEnabled ?? false,
+            pendingAction: policy?.pendingFields?.areConnectionsEnabled,
+            action: (isEnabled: boolean) => {
+                Policy.enablePolicyConnections(policy?.id ?? '', isEnabled);
+            },
+        },
+    ];
```

Comment:
Let's remove these until the connections settings is implemented
```suggestion
    const integrateItems: Item[] = [];
```